### ISSUE TITLE
SmartDate example

### ISSUE THREAD

**Issue by Chicagoan2016** [MarimerLLC/cslaforum#388]
Do we have a sample where SmartDate is used?

Kind Regards

**Comment by rockfordlhotka** [comment]
It is used in the [ProjectTracker sample](https://github.com/MarimerLLC/csla/tree/master/Samples/ProjectTracker)

**Comment by Chicagoan2016** [comment]
My mistake! it's in ResourceAssignmentEdit.cs.
Thank you Rocky

Regards

**Comment by Chicagoan2016** [comment]
When we send a SmartDate parameter  e.g. m_DOB to a stored proc that's expecting datetime argument, should we send m_DOB.DBValue ? (that's what I was doing back in the days of Csla 1.x).

And when I would retrieve a datetime field I would do something like
  m_DOB = .GetSmartDate("DOB", m_DOB.EmptyIsMin) 

kind regards

**Comment by tfreitasleal** [comment]
Hi,

I don't intend to "sell" you CslaGenFork as a production tool (although you can use to generate production code). But this kind of small questions is the main reason I took over CslaGen and made CslaGenFork. "Why re-invente the wheel (aka search/test) especially the small wheels..."

I use CslaGenFork as a repository of how to do things. To answer that kind of questions I would go to CslaGenFork create an object on the fly with just two properties: ID and a DateTime. I could use any DB table that includes columns wit the appropriate data types. Then F5 and have a look at the generated code.

I know that's not the "fish" it just my way a "fishing". Sorry if it didn't help.

**Comment by Chicagoan2016** [comment]
@tfreitasleal , I did download CslaGenFork a while ago (version 4.5.4) but didn't make a whole lot of progress with it because of lack of documentation/tutorial ( at that time), Could you please forward me a link to some kind of 'getting started' tutorial?

Regards

**Comment by tfreitasleal** [comment]
You're right. I never got the time to build a proper how to page. The closest thing is 
[CslaGen Wiki](http://cslagen.wiki-site.com/index.php/Main_Page)
